app/src/analytics/analytics-client.ts

```diff
@@ -0,0 +1,91 @@
+import crypto from 'crypto';
+
+interface AnalyticsAttributes {
+    [key: string]: any;
+}
+
+interface EventPayload {
+    eventName: string,
+    action: string,
+    actionSubject: string,
+    attributes?: AnalyticsAttributes
+}
+
+enum EnvType {
+    LOCAL = 'local',
+    DEV = 'dev',
+    STAGING = 'staging',
+    PROD = 'prod'
+}
+
+// eslint-disable-next-line max-len
+export const sendAnalytics = async (cloudId: string, eventPayload: EventPayload, accountId?: string, anonymousId?: string): Promise<void> => {
```

Comment:
wrapping up the actual sendEvent function with a .then .catch so we don't have to await it, this way the code doesnt wait for analytics to send before continuing. Also if it fails for any reason its swallowed up instead of breaking things.

Reply:
Nice - I like this!